### ISSUE TITLE
Remove "Riot is now Element"

### ISSUE THREAD

**Issue by notramo** [vector-im/element-android#2431]
The rebranding has been done months ago, and everybody who used Riot kows that it is called Element now.
**Having Riot mentioned everywhere is just confusing for new users.** It's complicated enough that there is Matrix and Element, and what's the difference.
The "Riot is now Element" dialog that is displayed after upgrading an installed Riot database should remain.

**Comment by LongJohn-Silver** [comment]
Yeah, I also hate that it says Element (Riot.im), I think  it should go with Element alone now.
